adr.md

```
@@ -0,0 +1,68 @@
+# ADR-334a49bf: Resolve match_same_arms Warnings in preprocess.rs
+
+## Status
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)